charts/ratify/templates/regopolicy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "ratify.fullname" . }}-rego-policy
+data:
+  policy.rego: |
+    package ratify.policy
+    default valid := false
+    valid {
+      not failed_verify(input)
+    }
+    failed_verify(reports) {
+      [path, value] := walk(reports)
+      value == false
+      path[count(path) - 1] == "isSuccess"
```

Comment:
As discussed, lets add comments if possible. On this specific verification, how are we validating isSuccess is true?